src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -2585,6 +2600,28 @@ private static void RemoveLastNullCompletionResult(List<CompletionResult> result
             }
         }
 
```

Comment:
`Get-Verb` completion should also take account `-Group` parameter.

Reply:
Thanks @ThomasNieto. I've also included code to take `-Group` into account. Let me know what you think 🙂.